pulp_ansible/app/tasks/collections.py

```diff
@@ -179,6 +179,17 @@ def sync(remote_pk, repository_pk, mirror, optimize):
     d_version = AnsibleDeclarativeVersion(first_stage, repository, mirror=mirror)
     repo_version = d_version.create()
 
+    last_sync_task = (
+        Task.objects.filter(
+            reserved_resources_record__contains=repository.pk, name__contains="sync"
+        )
+        .order_by("-pulp_created")
+        .first()
+    )
```

Comment:
```suggestion
    last_sync_task = Task.current()
```
We have a special method for finding the task object of the current executing task!

Reply:
okay, that's cool :+1: